### DISCUSSION TITLE
Be able to set Numbro.js language

### DISCUSSION THREAD

**Discussion by gerardnll** [laravel/nova-issues#5272] [Ideas & Feature Requests]
Seems that the library is available in multiple languages but I don't find the way to set it on Nova.

**Comment by davidhemphill** [comment]
I believe this is possible by customizing the `meta.blade.php` partial and adding a `meta` tag with the locale you wish to use: 

```html
<meta name="locale" content="de-DE" />
```

I'm curious, though, what are you trying to do client-side that you need this?

**Comment by gerardnll** [comment]
I changed the format of some Trends. They now show 1,000 (USA) instead of 1.000 (Europe) when indicating thousands.

**Comment by crynobone** [comment]
You should be able to define user locale using the following instead of modifying `meta.blade.php`:

https://nova.laravel.com/docs/4.0/customization/localization.html#user-locale-overrides
